cartography/intel/aws/ec2/internet_gateways.py

```diff
@@ -22,6 +23,18 @@ def get_internet_gateways(boto3_session: boto3.session.Session, region: str) ->
         client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
         internet_gateways = client.describe_internet_gateways()['InternetGateways']
 
+        default_vpc = get_default_vpc(client)
+        default_vpc_id = default_vpc.get('VpcId') if default_vpc else None
+
+        for igw in internet_gateways:
```

Comment:
do we get internet gateways by default? if not, this coudl be tricky.

let's say i create an internet gateway in a default vpc, that does not mean the internet gateway is default. it's still user created.

Reply:
So yes there's only one single default internet gateway created for a VPC. Changes I made are:

We now consider an Internet Gateway as 'predefined' only if:
    - It's attached to the default VPC (By matching the VPC IDs) and
    - It was created at approximately the same time as the default VPC (within 1 minute, kept a buffer of 1 min)